src/widget/form/profileform.cpp

```diff
@@ -247,10 +247,14 @@ void ProfileForm::onRenameClicked()
     Nexus& nexus = Nexus::getInstance();
     QString cur = nexus.getProfile()->getName();
     QString title = tr("Rename \"%1\"", "renaming a profile").arg(cur);
+
     do
     {
-        QString name = QInputDialog::getText(this, title, title+":");
-        if (name.isEmpty()) break;
+        QString name = QInputDialog::getText(0, title, title + ":");
```

Comment:
> You will get to the problem when using more than one monitor. You pretty much always want to give a dialog a parent

New dialog is always on parent window, even if parent is set 0. Never had this problem you described.